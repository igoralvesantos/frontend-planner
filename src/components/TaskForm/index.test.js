import React from "react";
import renderer from "react-test-renderer";
import { TaskForm } from "./index";

it("Snapshot TaskForm", () => {
  const tree = renderer
  .create(<TaskForm day="" text="" onChange={()=>{}} onSubmit={()=>{}}></TaskForm>)
  .toJSON();
  expect(tree).toMatchSnapshot();
});
