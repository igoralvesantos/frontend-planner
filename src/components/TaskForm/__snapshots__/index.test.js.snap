// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot TaskForm 1`] = `
<div
  className="sc-bdVaJa feQcXu"
>
  <form
    className="rs-form rs-form-inline rs-form-fixed-width"
    onSubmit={[Function]}
  >
    <input
      className="sc-bwzfXH JkfSJ"
      name="text"
      onChange={[Function]}
      placeholder="Digite uma tarefa"
      required={true}
      value=""
    />
    <select
      className="sc-htpNat iLAdUY"
      name="day"
      onChange={[Function]}
      required={true}
      value=""
    >
      <option
        disabled={true}
        selected={true}
        value=""
      >
        Dia da Semana
      </option>
      <option
        value="Segunda"
      >
        Segunda-Feira
      </option>
      <option
        value="Terça"
      >
        Terça-Feira
      </option>
      <option
        value="Quarta"
      >
        Quarta-Feira
      </option>
      <option
        value="Quinta"
      >
        Quinta-Feira
      </option>
      <option
        value="Sexta"
      >
        Sexta-Feira
      </option>
      <option
        value="Sábado"
      >
        Sábado
      </option>
      <option
        value="Domingo"
      >
        Domingo
      </option>
    </select>
    <button
      className="rs-btn rs-btn-primary rs-btn-md"
      onClick={[Function]}
      type="button"
    >
      Criar
      <span
        className="rs-ripple-pond"
      >
        <span
          className="rs-ripple"
          style={Object {}}
        />
      </span>
    </button>
  </form>
</div>
`;
